docker-compose.test.yml

```diff
@@ -47,7 +47,7 @@ services:
       - BCDA_ENABLE_NEW_GROUP=true
       - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993,A9994
     volumes:
-      - .:/go/src/github.com/CMSgov/bcda-app
+      - ./test_results:/go/src/github.com/CMSgov/bcda-app/test_results
```

Comment:
The spillage is acceptable, I think.  I believe we originally set it up this way so that a developer could make changes to a test file locally, and then running `make unit-test` would pick up those changes.  When I pulled down this branch,  and ran `make unit-test` after making a change to a test file, it did not pick up my changes.

Reply:
Ah, that's fair point. Since we mounted the entire directory, when we spun up the container, we would have the entire directory available to us. I'm going to update the Makefile to rebuild the image so we can pick up local changes. That way we can pick up local changes (as expected) and still allow the Docker container to be self contained (i.e. using resources when the image was built).

Reply:
Good idea!